File: src/main/java/main/after_startup/DeletingAllSlashCommandsFromAllGuilds.java
package main.after_startup;

import main.Main;
import net.dv8tion.jda.api.entities.Guild;

public class DeletingAllSlashCommandsFromAllGuilds implements IAfterStartUp {
    @Override
    public void doAfterStartUp() {
        Main.publicJDA.getGuilds().forEach(Guild::updateCommands);
    }
}

File: src/main/java/main/after_startup/DeletingAllSlashCommandsFromAllGuilds.kt
package main.after_startup

import main.StartUp

class DeletingAllSlashCommandsFromAllGuilds : IAfterStartUp {
    override fun doAfterStartUp() {
        // SlashCommandManager slashCommandManager = new SlashCommandManager();
        for (guild in StartUp.publicJDA.guilds) {
            guild.updateCommands()
        }
    }
}